spelling-allowlist.txt

```diff
@@ -106,5 +106,7 @@ Appender
 desc
 astype
 nullable
+SuiteResultWidgetSerializer
+detalize
```

Comment:
what is detalize?

Reply:
oh, yeap, it probably should be `detailise` - to make something detail; to split into details;
name of function (`detailise_additional_output`) from `deepchecks.core.check_result`

actually, after looking in the eng dictionary I did not find its definition so I am not sure whether it is correct you use `details` like a verb in this way/ maybe will find another name for it